game/src/main/scala/io/aigar/game/Cell.scala

```diff
@@ -66,6 +67,11 @@ class Cell(id: Int, startPosition: Vector2 = new Vector2(0f, 0f)) {
     return if (dir.magnitude > 0) dir.normalize * value else new Vector2(0f,0f)
   }
 
+  def contains(pos: Vector2): Boolean = {
+    return position.distanceTo(pos) <= mass
```

Comment:
A question about the intention not the code. Do we really want to completely be over the other cell to be able to eat it? In agar.io I don't think it's that far.